config/software/z_calibration.cfg

```diff
@@ -20,15 +20,10 @@ switch_xy_position: -1,-1
 # Switch offset (default D2F-5: 0.5mm and SSG-5H: 0.7mm)
 # Smaller number means higher nozzle to the PEI
 switch_offset: 0.50
-max_deviation: 0.750
+offset_margins: -0.750, 1.0
```

Comment:
does that work? `-0.750, 1.0`

Reply:
Yes, in the last version of the plugin (>v1.0.x) the `max_deviation` is deprecated in favor of the newly added `offset_margins` parameter.

FYI, currently the Z calibration plugin is broken since 2 weeks in Klippain due to this change (solved by replacing the full file in overrides, so not a big deal but still)